pkg/ring/lifecycler.go

```diff
@@ -778,6 +781,16 @@ func (i *Lifecycler) SetFlushOnShutdown(flushOnShutdown bool) {
 	i.flushOnShutdown.Store(flushOnShutdown)
 }
 
+// SkipUnregister returns if unregistering is skipped on shutdown.
```

Comment:
**suggestion:**

```suggestion
// UnregisterFromRing returns if unregistering is skipped on shutdown.
```

Reply:
Do'h, I really thought I renamed that too... Fixed.